src/cow-react/modules/limitOrders/pure/Orders/index.tsx

```diff
@@ -6,7 +6,7 @@ export interface OrdersProps extends OrdersTabsProps, OrdersTableProps {
   isWalletConnected: boolean
 }
 
-export function Orders({ orders, tabs, onTabChange, isWalletConnected }: OrdersProps) {
+export function Orders({ orders, tabs, isWalletConnected }: OrdersProps) {
   const content = () => {
```

Comment:
I see you really like this pattern :) 

I guess this is ultimatelly because you don't want to use a a `let` ever. For me in small components, as long as its easy to reason about it.

Also, in this case there could be a subcomponent defined as:

`export function OrdersContent(props: OrdersProps) {}`

Anyways, all options are fine with me. This looks great!

Reply:
>in this case there could be a subcomponent defined as

It's a good idea, will use it, thanks!